src/Server.ts

```diff
@@ -35,6 +38,16 @@ export class Server {
 
   private createContainerWithBindings(): Container {
     const container = new Container();
+    container.bind<RedisService>(TYPES.RedisService).to(RedisService);
```

Comment:
Since more and more things were added to the container I suggest you move it off the `Server.ts` cause in current shape it has too many responsibilities reasons to change. It definitely brakes Single Responsibility Principle.

Reply:
Agreed 👍

Reply:
Instead of binding to the container in this way, we could use decorators: https://github.com/inversify/inversify-binding-decorators

Reply:
Yes that's much nicer thanks!